src/Orckestra.Composer.Cart/ViewModels/Order/OrderShipmentDetailViewModel.cs

```diff
@@ -70,6 +71,9 @@ public sealed class OrderShipmentDetailViewModel : BaseViewModel
         /// Gets or sets the Shipping method viewmodel for this shipment.
         /// </summary>
         public OrderShippingMethodViewModel ShippingMethod { get; set; }
+        public Guid FulfillmentLocationId {get; set; }
```

Comment:
Please move this as well:
![image](https://user-images.githubusercontent.com/6649972/94685378-0447e200-0332-11eb-8c3a-c3e6f89898b9.png)